packages/dashboard/src/pages/Labs/components/SwapSettingDialog.tsx

```diff
@@ -65,6 +71,12 @@ export default function SwapSettingDialog({ open, onClose }: SettingDialogProps)
             options: polygonOptions,
             onChange: (value: string) => Services.Settings.setPolygonNetworkTradeProvider(Number.parseInt(value, 10)),
         },
+        {
+            legend: t.labs_settings_swap_network({ network: 'Fantom' }),
+            value: useFantomNetworkTradeProvider(),
+            options: fantomOptions,
+            onChange: (value: any) => Services.Settings.setFantomNetworkTradeProvider(+value),
```

Comment:
```suggestion
            onChange: (value: string) => Services.Settings.setFantomNetworkTradeProvider(Number.parseInt(value, 10)),
```

Reply:
@Jack-Works can simplify the code?

```suggestion
            onChange: Services.Settings.setFantomNetworkTradeProvider,
```